### PR TITLE
Adding changes for Quarkus 3 Beta

### PR DESCRIPTION
**Pull request by harshmadhani**
Adding changes for Quarkus 3 Beta

### CODE REVIEW

File: scm-info/com/google/api/grpc/scm.yaml [hunk 1]
```diff
@@ -1,3 +1,6 @@
 ---
 type: "git"
-uri: "https://github.com/googleapis/java-common-protos.git"
+uri: "https://github.com/googleapis/gapic-generator-java.git"
+tagMapping:
+  - pattern: 2.14.3
```
[the remark this answers is not in the document]
Comment (stuartwdouglas) [reply scm-info/com/google/api/grpc/scm.yaml:5]: Sorry, I forgot to hit submit on this comment last week, and it was stuck in pending.

[the remark this answers is not in the document]
Comment (stuartwdouglas) [reply scm-info/com/google/api/grpc/scm.yaml:5]: But why is 2.14 being mapped to 2.15?

[the remark this answers is not in the document]
Comment (stuartwdouglas) [reply scm-info/com/google/api/grpc/scm.yaml:5]: Ok, thanks, I was just checking because it seemed odd.

[the remark this answers is not in the document]
Comment (harshmadhani) [reply scm-info/com/google/api/grpc/scm.yaml:5]: Yeah, because it is odd :)

**Pull request merged**